css/infobar.css

```diff
@@ -44,3 +49,7 @@
 .mozilla-infobar [dir="rtl"] li {
     margin: 0 0 0 10px;
 }
+
+.hidden {
```

Comment:
This is a very generic and common selector. Suggest name-spacing to avoid unintended breakage in other sites?